lib/eslintrc.json

```diff
@@ -14,136 +14,17 @@
   "plugins": [
     "flowtype",
     "import",
-    "jest",
-    "jsx-a11y"
+    "jest"
   ],
   "rules": {
-    "complexity": ["warn", 10],
+    "complexity": ["warn", 12],
     "import/order": ["warn", {
       "newlines-between": "always"
     }],
-    "no-magic-numbers": ["warn", {
-      "ignore": [-1, 0, 1, 2]
-    }],
     "prefer-const": ["warn", {
       "destructuring": "all",
       "ignoreReadBeforeAssign": false
-    }],
-    "require-jsdoc": ["warn", {
-      "require": {
-        "FunctionDeclaration": true,
-        "MethodDefinition": true,
-        "ClassDeclaration": true
-      }
-    }],
-    "sort-keys": "warn",
-    "sort-vars": "warn",
```

Comment:
I think these sort plugins are worth keeping.  I like having orderly variables.

Reply:
I agree for the 90% case. The problem is when you don't want them sorted alphabetically for a specific reason (it happens) and then it's a warning or must be commented to be ignored. I think that extraneous warning/comment outweighs the benefits. Something that should be in a "style guide" but not done by a linter.